docs/DeveloperGenerateTest.md

```diff
@@ -174,6 +174,39 @@ Before generating anything make sure you have Ginkgo in place. To install Ginkgo
 
 `go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo`
 
+## Team specific template
+Teams can create their own templates according to their needs.
+These should utilize the provided `specs.tmpl` by including the following line wherever they want their specs to be. (do not forget the dot!) 
+
+`{{ template "specs" . }}`
+
+Please see the provided example:
+```
+package {{ .CustomData.PackageName }}
```

Comment:
@tnevrlka could we use a fuller real example here? Like maybe utilize a modified an existing test file the has the BeforeSuite/AfterSuite and imports.